src/SWP/Bundle/TemplatesSystemBundle/DependencyInjection/ContainerBuilder/MetaLoaderCompilerPass.php

```diff
@@ -31,13 +31,23 @@ public function process(ContainerBuilder $container)
         );
 
         $taggedServices = $container->findTaggedServiceIds(
-            'swp.meta_loader'
+            'swp.meta_loader.add'
         );
         foreach ($taggedServices as $id => $tags) {
             $definition->addMethodCall(
                 'addLoader',
                 [new Reference($id)]
             );
         }
+
+        $taggedServices = $container->findTaggedServiceIds(
+            'swp.meta_loader.remove'
```

Comment:
it looks that there are no services registered with this tag, what is it used for?